src/averageAges.js

```diff
@@ -15,6 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const centuryLived = people.filter(person => {
+    if (century) {
+      return person.sex === 'm' && Math.ceil(person.died / 100) === century;
+    } else {
+      return person.sex === 'm';
+    };
+  });
+
+  const summedAges = centuryLived.map(person => person.died - person.born)
+    .reduce((prev, item) => prev + item, 0);
```

Comment:
Let's use here only reduce, without `map`

Reply:
yep, now it looks obviouos)
but it`s only after 'js practice' and tehCheck)
done